scripts/deploy-stonks-factory.ts

```diff
@@ -0,0 +1,77 @@
+import { ethers, network } from 'hardhat'
+
+import fmt from '../utils/format'
+import { confirmOrAbort } from '../utils/prompt'
+import { StonksFactory__factory } from '../typechain-types'
+import { getDeployer, verify } from '../utils/deployment'
+import { OrderSampleDeployedEvent } from '../typechain-types/contracts/factories/StonksFactory'
+
+const AGENT = ''
+const SETTLEMENT = ''
+const VAULT_RELAYER = ''
```

Comment:
Those names are a bit generic to remember their meaning

Reply:
maybe:
`COWSWAP_SETTLEMENT` and `COWSWAP_VAULT_RELAYER`? (following the same pattern for `CHAINLINK_PRICE_FEED_REGISTRY`)